sunpy/net/jsoc/attrs.py

```diff
@@ -3,17 +3,22 @@
 import astropy.units as u
 
 from sunpy.net.attr import (Attr, AttrWalker, AttrAnd, AttrOr)
-from sunpy.net.vso.attrs import Time, _VSOSimpleAttr
+from sunpy.net.vso.attrs import Time,Sample,_VSOSimpleAttr
 
 __all__ = ['Series', 'Protocol', 'Notify', 'Compression', 'Wavelength', 'Time',
-           'Segment']
+           'Segment','Sample']
 
 
 class Time(Time):
     """
     Time range to download
     """
-
+    
+class Sample(Sample):
```

Comment:
You don't need this here, just the import from vso will suffice, you will still be able to do `jsoc.attrs.Sample`